content/docs/intro/cloud-providers/docker/setup.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Docker Setup
+meta_desc: This page provides an overview on how to configure the Pulumi Docker Provider.
+aliases: ["/docs/reference/clouds/docker/setup/"]
```

Comment:
Similarly, this line should be removed. The alias isn't needed for brand new pages (only when moving existing pages to different paths).

Reply:
done!